boilerplate/whatwg/header-RD.include

```diff
@@ -15,7 +15,7 @@
   </a>
   <hgroup>
     <h1 id="title" class="p-name no-ref">[SPECTITLE]</h1>
-    <h2 id="w3c-state" class="no-num no-toc no-ref">[LONGSTATUS] — Published <time class="dt-updated" datetime="[ISODATE]">[DATE]</time></h2>
+    <p id="w3c-state" class="no-num no-toc no-ref">[LONGSTATUS] — Published <time class="dt-updated" datetime="[ISODATE]">[DATE]</time></p>
```

Comment:
not sure that the whatwg stylesheet support this change. Nor any of the wg14, wg21, tc39